hack/manifest-gen/assets/charts/eck/templates/operator-role-binding.yaml

```diff
@@ -0,0 +1,17 @@
+{{- $operatorNSIsManaged := has .Values.operator.namespace .Values.config.managedNamespaces -}}
+{{- if not $operatorNSIsManaged -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ .Values.operator.name }}
+  namespace: {{ .Values.operator.namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ .Values.operator.name }}
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.operator.name }}
+  namespace: {{ .Values.operator.namespace }}
+{{- end -}}
```

Comment:
isn't it done already in `managed-ns-role-bindings.yaml`? Why is this a special case?